src/js/bootstrap.js

```diff
@@ -1,63 +1,114 @@
+      /**
+       * Remove cross account request if it is expired
+       */
+      if (new Date(end_date).getTime() <= Date.now()) {
+        crossAccountBouncer();
+      }
+    } catch {
+      console.log('Unable to parse initial account. Using default account');
+      Cookies.remove(CROSS_ACCESS_ACCOUNT_NUMBER);
     }
-  } catch {
-    console.log('Unable to parse initial account. Using default account');
-    Cookies.remove(CROSS_ACCESS_ACCOUNT_NUMBER);
   }
-}
+};
+
+const libjwtSetup = () => {
+  const libjwt = auth();
+
+  function noop() {}
+
+  libjwt.initPromise.then(() => {
+    libjwt.jwt
+      .getUserInfo()
+      .then((...data) => {
+        sentry(...data);
+      })
+      .catch(noop);
+  });
+
+  return libjwt;
+};
+
+const trustarcScriptSetup = () => {
+  const trustarcScript = document.createElement('script');
+  trustarcScript.id = 'trustarc';
+
+  if (location.host === 'console.redhat.com') {
+    trustarcScript.src = '//static.redhat.com/libs/redhat/marketing/latest/trustarc/trustarc.js';
+  } else {
+    trustarcScript.src = '//static.redhat.com/libs/redhat/marketing/latest/trustarc/trustarc.stage.js';
+  }
 
-// start auth asap
-const libjwt = auth();
+  document.body.appendChild(trustarcScript);
+};
 
-function noop() {}
+const App = () => {
+  const modules = useSelector(({ chrome }) => chrome?.modules);
+  const scalprumConfig = useSelector(({ chrome }) => chrome?.scalprumConfig);
+  const documentTitle = useSelector(({ chrome }) => chrome?.documentTitle);
+  const dispatch = useDispatch();
+  const [jwtState, setJwtState] = useState(false);
 
-//Add redhat font to body
-document.querySelector('body').classList.add('pf-m-redhat-font');
+  useEffect(() => {
+    initializeAccessRequestCookies();
+    const libjwt = libjwtSetup();
+    libjwt.initPromise.then(() => setJwtState(true));
 
-libjwt.initPromise.then(() => {
-  libjwt.jwt
-    .getUserInfo()
-    .then((...data) => {
-      sentry(...data);
-    })
-    .catch(noop);
-});
+    window.insights = createChromeInstance(libjwt, window.insights);
 
-window.insights = window.insights || {};
+    if (typeof _satellite !== 'undefined' && typeof window._satellite.pageBottom === 'function') {
+      window._satellite.pageBottom();
+      registerUrlObserver(window._satellite.pageBottom);
+    }
 
-window.insights = createChromeInstance(libjwt, window.insights);
+    trustarcScriptSetup();
 
-if (typeof _satellite !== 'undefined' && typeof window._satellite.pageBottom === 'function') {
-  window._satellite.pageBottom();
-  registerUrlObserver(window._satellite.pageBottom);
-}
+    axios
+      .get(`${window.location.origin}${isBeta() ? '/beta' : ''}/config/chrome/fed-modules.json?ts=${Date.now()}`, {
+        headers: {
+          'Cache-Control': 'no-cache',
+          Pragma: 'no-cache',
+          Expires: '0',
+        },
+      })
+      .then((response) => {
+        dispatch(loadModuesSchema(response.data));
```

Comment:
Since we're refactoring this file I'd fix the type on this function
```suggestion
        dispatch(loadModulesSchema(response.data));
```